test/extended/util/annotate/rules.go

```diff
@@ -166,7 +166,7 @@ var (
 			`\[sig-instrumentation\] Prometheus when installed on the cluster when using openshift-sdn should be able to get the sdn ovs flows`,
 			`\[sig-instrumentation\]\[Late\] Alerts should have a Watchdog alert in firing state the entire cluster run`,
 			`\[sig-instrumentation\]\[Late\] Alerts shouldn't exceed the 500 series limit of total series sent via telemetry from each cluster`,
-			`\[sig-instrumentation\]\[Late\] Alerts shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured`,
+			`\[sig-instrumentation\]\[Late\] Alerts shouldn't report any alerts in firing or pending state apart from Watchdog and AlertmanagerReceiversNotConfigured and have no gaps in Watchdog firing`,
```

Comment:
Are we still firing alerts in disconnected mode?  Or is this because we can't access the test?  This is such a fundamental safety net that the fact that we shipped 4.8 with this failing means we have a huge knowledge gap as to whether disconnected is safe.

Is there a story covering reenabling this test because "we've fixed this"?  I see there's a comment up above for "ICSP" - isn't that bug fixed?  In the future, please add bugzilla links to these sections that have a bug we can track for "this problem is now fixed".